apiary.apib

```diff
@@ -39,7 +40,42 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             },
         ]
 
-## CodeSchool | Create [/api/v1/code_schools{?name,url,logo,full_time,hardware_included,has_online,online_only,notes}]
+## MOOC CodeSchool | Collection [/api/v1/code_schools/get_moocs]
+
+### List all MOOCS [GET]
+
++ Response 200 (application/json)
+
+        [
```

Comment:
Good question. Do you know who is working this on the front end? I checked out the linked issue, but looks like there hasn't been chatter since Oct.

Reply:
@leenyburger - Let's have this be in keeping with the structure that the current [CodeSchools - Collection](https://operationcodeapi.docs.apiary.io/#reference/0/codeschool-collection/list-all-codeschool-members) endpoint returns.

cc: @kylemh

Reply:
It's a low priority issue to be sure since we don't add MOOCs often.

I haven't seen this issue on the back-end, but I would prefer that MOOCs not need to be a separate API call. Instead have an `is_mooc` boolean for all existing code schools should be sufficient.